o2a/utils/el_utils.py

```diff
@@ -208,38 +114,65 @@ def comma_separated_string_to_list(line: str) -> Union[List[str], str]:
     return values[0] if len(values) <= 1 else values
 
 
+def _resolve_name_node(translation: str, props: PropertySet) -> Optional[str]:
+    """
+    Check if props include nameNode, nameNode1 ro nameNode2 value.
+    """
+    merged = props.merged
+    for key in ["nameNode", "nameNode1", "nameNode2"]:
+        start_str = "{{" + key + "}}"
+        name_node = merged.get(key)
+        if translation.startswith(start_str) and name_node:
+            return name_node
+    return None
+
+
 def normalize_path(url: str, props: PropertySet, allow_no_schema=False) -> str:
     """
-    Replaces all EL variables in the url, validates schema and returns only the 'path' part of a url.
-    Example: hdfs://localhost:8082/user/root --> user/root
+    Translates url to jinja template. Validates schema using `nameNode` value from props.
```

Comment:
What does it mean to "translate to jinja template" in this context? I think a bit more explanation is needed on what's going on here - it's not really obvious from the code